e2e_test/udf/sql_udf.slt

```diff
@@ -57,12 +57,12 @@ create function fib(INT) returns int
       end;';
 
 # The execution will eventually exceed the pre-defined max stack depth
-statement error function fib calling stack depth limit exceeded
-select fib(100);
+# statement error function fib calling stack depth limit exceeded
```

Comment:
Why it's commented?

Reply:
> Note: This will also prevent recursion from happening, future support to recursion may need extra work around.

Since we don't allow users to create recursion function if semantic check is enabled.